packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -349,10 +349,15 @@ async function createOutgoingPayment(
             description: 'Time to retrieve peer in outgoing payment'
           }
         )
-        const peer = await deps.peerService.getByDestinationAddress(
-          receiver.ilpAddress,
-          tenantId
+        const ilpPaymentMethod = receiver.paymentMethods.find(
+          (method) => method.type === 'ilp'
         )
+        const peer = ilpPaymentMethod
+          ? await deps.peerService.getByDestinationAddress(
+              ilpPaymentMethod.ilpAddress,
+              tenantId
+            )
+          : undefined
```

Comment:
We only need this to store a `peerId` on the outgoingPayment, even though we don't use it for anything